tags.go

```diff
@@ -33,16 +33,16 @@ type TagsService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/tags.html
 type Tag struct {
-	Commit  *Commit  `json:"commit"`
-	Release *Release `json:"release"`
-	Name    string   `json:"name"`
-	Message string   `json:"message"`
+	Commit  *Commit     `json:"commit"`
+	Release *TagRelease `json:"release"`
+	Name    string      `json:"name"`
+	Message string      `json:"message"`
 }
 
-// Release represents a GitLab version release.
+// TagRelease represents a GitLab version release.
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/tags.html
-type Release struct {
+type TagRelease struct {
```

Comment:
I think I like `ReleaseTag` better. To me is sounds and reads a bit nicer. What do you think, would it fit this use case or is `TagRelease` better?

Reply:
I don't know. I agree that ReleaseTag sounds better, but maybe TagRelease still describes the purpose more accurately.

For example CreateReleaseTag suggests that it creates a special kind of tag; CreateTagRelease is closer to the truth.

I think TagDescription, ReleaseDescription and ReleaseNote would also describe the purpose quite accurately.

Actually, I give my vote to ReleaseNote, because it's the term that is used in gitlab documentation. https://docs.gitlab.com/ce/api/tags.html#create-a-new-release

Reply:
I'm all good with `ReleaseNote` 👍 Go for it 😉